Dockerfile

```diff
@@ -1,15 +1,18 @@
-FROM node:18-bullseye-slim
+FROM node:20-slim
 WORKDIR /app
+EXPOSE 9000
 
-RUN apt-get update
```

Comment:
`apt-get update` is not redundant - at worst, it does nothing and at best it makes the package build successfully

Reply:
https://stackoverflow.com/a/63901529/9878135

Reply:
from your link:
> You're somewhat dependent on intermediate images, like python:3.8, doing the same thing.
is it documented anywhere that the official `node` package does this? i tried to look for it but couldn't find any official mention of it.

Reply:
From https://hub.docker.com/_/node/:

> Some of these tags may have names like bookworm, bullseye, or buster in them. These are the suite code names for releases of [Debian](https://wiki.debian.org/DebianReleases) and indicate which release the image is based on

So it's based on Debian - which I assume does update its packages..

Reply:
that's still a bit too ambiguous for my liking, so i would probably prefer to keep `apt-get update` in there